iam/terraform/account-specific/main/lambda-es-role.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_iam_role" "lambda_elasticsearch_execution_role" {
+  name = "lambda_elasticsearch_execution_role"
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "lambda_elasticsearch_execution_policy" {
+  name = "lambda_elasticsearch_execution_policy"
+  role = aws_iam_role.lambda_elasticsearch_execution_role.id
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": [
+        "arn:aws:logs:*:*:*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": "es:ESHttpPost",
+      "Resource": "arn:aws:es:*:*:*"
+    }
+  ]
+}
+EOF
+}
```

Comment:
These two resources seem unused, unless I’m missing where they’re used. It seems to be allowing lambdas to write cloudwatch logs and also post things to any elasticsearch instance?

Reply:
You were right that we didn't need them previously, but now that we have configuration for log subscriptions, this role/policy will allow the LogsToElasticsearch_info lambda to publish logs to Cloudwatch.

Reply:
I removed the ES permission, I don't think that is needed: https://github.com/ustaxcourt/ef-cms/pull/387/commits/288e626fd9ab0db16564ee52df044a3806bf5827